FetchRequests/Sources/Associations/ObservableToken.swift

```diff
@@ -120,7 +148,9 @@ internal class LegacyKeyValueObserving<Object: NSObject, Value: Any>: NSObject,
 }
 
 internal class FetchRequestObservableToken<Parameter>: ObservableToken {
-    private let _observe: (_ handler: @escaping (Parameter) -> Void) -> Void
+    typealias Handler = (Parameter) -> Void
```

Comment:
Making this @MainActor (which requires making ObservableToken's handler @MainActor) was a giant pain and not worth it since I could do the simple main-thread bouncing behavior.